batchflow/models/tf/mobilenet.py

```diff
@@ -134,31 +161,46 @@ class MobileNet_v2(TFModel):
         parameters for the initial block (default is 'cna', 32, 3, strides=2)
 
     body : dict
-        layout : list of dict
-            a sequence of block parameters:
+        layout : list of dict each consisting of parameters:
             repeats : int
+                number of repeats of the block
             filters : int
+                number of output filters in the block
             expansion_factor : int
+                multiplier for the number of filters in internal convolutions
             strides : int
-
+                stride for 3x3 convolution
+            kernel_size : int
+                kernel size for depthwise convolution
+            activation : callable, optional
+            se_block : bool or dict
+                whether to include squeeze and excitation block
+                If dict, it must contain :meth:`~TFModel.se_block` params
+            residual : bool
+                whether to make a residual connection
         width_factor : float
             multiplier for the number of channels (default=1)
+
+    head : dict
+        se_block : dict, optional
+            None or a dict with :meth:`~TFModel.se_block` params
     """
     @classmethod
     def default_config(cls):
         config = TFModel.default_config()
-        config['common'].update(dict(activation=tf.nn.relu6))
         config['initial_block'].update(dict(layout='cna', filters=32, kernel_size=3, strides=2))
         config['body'].update(dict(width_factor=1, layout=deepcopy(_V2_DEFAULT_BODY)))
         config['head'].update(dict(layout='cnacnV', filters=[1280, 2], kernel_size=1))
-
+        config['common'].update(dict(activation=tf.nn.relu6))
         config['loss'] = 'ce'
-
         return config
 
     def build_config(self, names=None):
         config = super().build_config(names)
-        config['head']['filters'][-1] = self.num_classes('targets')
+        if isinstance(config['head/filters'], list):
+            config['head/filters'][-1] = self.num_classes('targets')
+        else:
+            config['head/filters'] = self.num_classes('targets')
```

Comment:
I though about this earlier and decided not to change it since this is needed for some models but not for others.
However, now `head` takes `targets` as a parameter so we can move this into `TFModel.head` and other model can easily redefine this behaviour when required.